doc/api/http2.md

```diff
@@ -1064,7 +1064,7 @@ const server = http2.createServer();
 server.on('stream', (stream) => {
   function statCheck(stat, headers) {
     // Check the stat here...
-    stream.respond({ ':status': 304 });
+    stream.respond({ ':status': 304 }, { endStream: true });
```

Comment:
FYI... the respond method will now automatically default `endStream` to `true` if the status code is `304`. See: https://github.com/nodejs/http2/blob/master/lib/internal/http2/core.js#L1651-L1659